c++/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 project("Cap'n Proto" CXX)
 cmake_minimum_required(VERSION 3.1)
-set(VERSION 0.7-dev)
+set(RELEASE_VERSION 0.7)
```

Comment:
The thing I was concerned about is that `0.6` and `0.6.1`. Both of those are supposed to have the same version at the library level.

The other thought I had was using `SOVERSION` and setting up the autotools like major.minor.patch variables and construct this properly, I just wasn't sure how your ABI guarantee worked, so I erred on the side of caution.

This is the variant I'm using in Fedora: http://pkgs.fedoraproject.org/cgit/rpms/capnproto.git/tree/capnproto-0.6.1-CMake-Add-versioning-to-the-libraries.patch